<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerComm</class>
 <widget class="QDialog" name="SerComm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SerComm</string>
  </property>
  <widget class="QTextEdit" name="textEdit_Rec">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Send">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>串口参数</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>串口号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_PortName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_OpenClose">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>开启</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_bandrate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Rec">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>接受：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Send">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>发送：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_RClear">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
